tools/walletextension/frontend/Dockerfile

```diff
@@ -21,25 +21,28 @@ WORKDIR /usr/src/app
 COPY tools/walletextension/frontend/ .
 
 # Install dependencies
-RUN pnpm ci
+RUN pnpm install --frozen-lockfile
 
 # Build the Next.js app
 RUN pnpm run build
 
 # Reduce the size of the final image by using a lighter base image
 FROM node:20-alpine AS runner
 
+# Install pnpm
+RUN npm install -g pnpm
```

Comment:
I think this one is redundant since its installed at the start? Not sure if that prod flag works the same with pnpm but maybe @pbienek might know more

Reply:
It is a different image (`base` and `runner` image)  and it needs `pnpm` installed. 
As far as I know `--prod` flag doesn't install dev dependencies which makes runner image smaller.